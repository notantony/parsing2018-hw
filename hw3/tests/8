f a b c =
if && (>= a b) (>= a c)
    (print a)
    (
        if && (>= b a) (>= b c)
            (print b)
            (print c)
    )