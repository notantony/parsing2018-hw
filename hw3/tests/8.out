void f(int a, int b, int c) {
    if ((((a >= b)) && ((a >= c)))) {
        std::cout << a;
}
else {
if ((((b >= a)) && ((b >= c)))) {
std::cout << b;
}
else {
std::cout << c;
}
}
}
